playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java

```diff
@@ -150,16 +155,22 @@
 
         playerSettings = settings != null ? settings : new PlayerSettings();
         rootView = rootPlayerView;
-        DefaultBandwidthMeter.Builder bandwidthMeterBuilder = new DefaultBandwidthMeter.Builder(context);
 
-        Long initialBitrateEstimate = playerSettings.getAbrSettings().getInitialBitrateEstimate();
+        LoadControlStrategy customLoadControlStrategy = (LoadControlStrategy) playerSettings.getCustomLoadControlStrategy();
```

Comment:
since this is sent as Object please add here validation that it is instance of  before casting else keep old logic